src/api/Search/types.ts

```diff
@@ -25,12 +25,12 @@ export interface GetIndexationStatusResponse {
 }
 
 export interface ISearchParams extends IDataSourceParams {
-  type: EntityType;
+  type?: EntityType;
   q: string;
   fuzziness?: number;
   size?: number;
   from?: number;
-  categoriesOrTypes?: string[];
```

Comment:
@pastahito since we have no choice but to change the API, rename the param to: categoryOrType: string.